maintenance.md

```diff
@@ -474,6 +474,16 @@ The libsystemd version should be the same with the one running on the host OS.
 5. Update ceph image tag in `Dockerfile`.
 6. Update `BRANCH` and `TAG` files.
 
+## sealed-secrets
+
+![Regular Update](./regular_update.svg)
+
+1. Check the [release page](https://github.com/bitnami-labs/sealed-secrets/releases).
+2. Check the upstream Dockerfile. If there are any updates, update our `Dockerfile`.
```

Comment:
It would be confusing to compre Dockerfiles because the upstream one does not contain the build stage.

```suggestion
2. Check the upstream Dockerfile and compare with ours especially on the runtime stage. If there are any updates, update our `Dockerfile`.
```